inventory/group_vars/zuul

```diff
@@ -17,3 +17,13 @@ bonnyci_zuul_apache_vhosts:
       ScriptAlias /p/ /usr/lib/git-core/git-http-backend/
 
 bonnyci_zuul_merger_git_dir: /var/lib/zuul/git
+
+logrotate_configs:
+  - name: zuul
+    path: /var/log/zuul/*log
+    options:
+      - compress
+      - missingok
+      - rotate 30
+      - daily
+      - notifempty
```

Comment:
I think this should be defined in zuul role defaults. Those can be used at role dependency time.